urdf2webots/parserURDF.py

```diff
@@ -422,7 +422,7 @@ def getOBJMesh(filename, node):
     with open(filename, 'r') as file:
         trimesh = node.geometry.trimesh
         for line in file:
-            header, body = line.split(maxsplit=1)
+            header, body = line.split(' ', 1)
```

Comment:
Just out of curiosity, what is the advantage of this change?

Reply:
Compatibility with Python 2.7